README.md

```diff
@@ -226,9 +226,9 @@ SAML_USERS_SYNC_ATTRIBUTES = True
 
 **SAML_USERS_STRICT_MAPPING (optional):**
 Specifies if every user attribute defined in SAML_USER_MAP must be present
-in the saml response or not. If set to False, ensure your Django user model
-defines those attributes which your IdP doesn't expose in the SAML response.
-You could do this by setting null=True or a default value in your user model.
+in the saml response or not. If set to False, you can specify a default
+value in the "SAML_USER_MAP" dict in case the attribute is not set in the
+IdP response.
 
```

Comment:
To make things a bit more obvious to users can we make the docs read like this?

Specifies whether, or not, every user attribute defined in SAML_USER_MAP must be present in the SAML response object. 

If set to False, you can optionally specify a default value in the "SAML_USER_MAP" dict will set the value when the attribute is not be present in the IdP response object. 

Example default value setting

```python
# set default value for is_superuser and is_staff to False
SAML_USERS_STRICT_MAPPING = False
SAML_USERS_MAP = [{
    "MyProvider" : {
      "email": dict(key="email", index=0),
      "username": dict(key="username", index=0),
      "is_superuser": dict(key="is_superuser", index=0, default=False),
      "is_staff": dict(key="is_staff", index=0, default=False)
    }
}]
```



```